.github/workflows/detect-breaking-changes-levitate.yml

```diff
@@ -1,5 +1,5 @@
 # Only runs if anything under the packages/ directory changes.
-
+---
```

Comment:
use the "ignore white spaces" option in github. I fixed the indentation in this file.